packages/cli/bin/console

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+namespace T3Docs\Cli;
+
+use T3Docs\Cli\Command\Console;
+use Symfony\Component\Console\Application;
+
+$vendorDir = dirname(__DIR__) . '/../../vendor';
+$autoloadDirectory = $vendorDir . '/autoload.php';
```

Comment:
$autoloadDirectory is not the correct variable name as this holds a file path, not a directory path.

Reply:
(We have the same code in the current bin/guides-cli, will check how this can be adapted)